src/components/modals/TradePreviewModal.vue

```diff
@@ -166,7 +166,7 @@ export default defineComponent({
 
     const tokenApproval = useTokenApproval(addressIn, amountIn, tokens);
 
-    const lidoRelayerApproval = useLidoRelayerApproval(isStETHTrade);
+    const lidoRelayerApproval = useRelayerApproval('lido', isStETHTrade);
```

Comment:
I've passed `useRelayerApproval` the responsibility of handling different relayers and their addresses.